Magento_Bundle/templates/catalog/product/view/type/bundle.phtml

```diff
@@ -0,0 +1,13 @@
+<?php $_product = $block->getProduct() ?>
+<?php if ($block->displayProductStockStatus()): ?>
+    <?php if ($_product->isAvailable()): ?>
+        <div class="product-view__stock-status">
+            <?= __('In stock') ?>
+        </div>
+    <?php else: ?>
+        <div class="product-view__stock-status product-view__stock-status--out-of-stock">
```

Comment:
to new line `class="product-view__stock-status product-view__stock-status--out-of-stock"`

Reply:
Moved